samtranslator/swagger/swagger.py

```diff
@@ -37,6 +37,7 @@ def __init__(self, doc):
         self.paths = self._doc["paths"]
         self.security_definitions = self._doc.get("securityDefinitions", {})
         self.gateway_responses = self._doc.get(self._X_APIGW_GATEWAY_RESPONSES, {})
+        self.definitions = self._doc.get('definitions', {})
```

Comment:
To make this work with Openapi 3, you would need to move definitions under $/components/schemas.
You could add it as a post process similar to the auth components here - https://github.com/awslabs/serverless-application-model/blob/develop/samtranslator/model/api/api_generator.py#L293

Here is a link on whats changed from swagger 2.0 to openapi 3.0 - 
https://blog.readme.io/an-example-filled-guide-to-swagger-3-2/

Reply:
@praneetap I've updated the PR to include openapi support.  Will you please have a look?

Reply:
The changes look good to me, have you tried to deploy an openapi transformed template manually on cloud formation? 
We would also really appreciate it if you add an example to the [examples](https://github.com/awslabs/serverless-application-model/tree/master/examples/2016-10-31)/.